classes/ActionScheduler_SystemInformation.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Provides information about active and registered instances of Action Scheduler.
+ */
+class ActionScheduler_SystemInformation {
```

Comment:
The code in this new class is essentially lifted direct from `ActionScheduler_Versions` (though they have become static methods in the process).